### PR TITLE
Updating participant information for 2023 October

### PR DESCRIPTION
**Pull request by Jen-OMalley**
Updated participant information for this workshop and made a few small tweaks elsewhere. This PR addresses [this issue](https://github.com/AlexsLemonade/training-admin/issues/1016), that was filed in `training-admin`.

### CODE REVIEW

File: docs/participant_information.md [hunk 1]
```diff
@@ -4,70 +4,67 @@ nav_title: Logistics
...
-  - [Nearby Lodging](#nearby-lodging)
 - [Travel Reimbursement](#travel-reimbursement)
-
--->
+- [Health and Safety](#health-and-safety)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-<!--
+## Action Items
```
Comment (sjspielman) [inline docs/participant_information.md:21]: Maybe, "Tasks to complete before the workshop"? I'm not sure "action items" conveys this as clearly?

File: docs/participant_information.md [hunk 2]
```diff
@@ -4,70 +4,67 @@ nav_title: Logistics
...
-* If you are not feeling well, please do not attend the workshop. 
-We will not withhold refunds if a participant must cancel because they are not feeling well. 
+**The workshop will be held at The Inn at Penn, a Hilton Hotel, located at [3600 Sansom St, Philadelphia, PA 19104](https://www.google.com/maps/place/The+Inn+at+Penn,+a+Hilton+Hotel/@39.9537829,-75.1983565,17z/data=!3m1!4b1!4m9!3m8!1s0x89c6c6575236ba05:0x2ab1dcf9ee499b6b!5m2!4m1!1i2!8m2!3d39.9537829!4d-75.1957816!16s%2Fg%2F1hc24w67l?entry=ttu)**. 
+We will meet in Conference Room 603. 
+If you have trouble finding the building or the room, please use Slack to direct message `Jen O'Malley`. 
 
-## Workshop Location
+The Inn at Penn is located in West Philadelphia in the [University City](https://www.universitycity.org/) area.
+The neighborhood is well-known for being home to the University of Pennsylvania and Drexel University.
+It is a largely walkable area with plenty of restaurants and attractions!
```
Comment (sjspielman) [inline docs/participant_information.md:35]: please consider this comment my vote for [Manakeesh](https://www.manakeeshcafe.com/) 🤤

File: docs/participant_information.md [hunk 3]
```diff
@@ -4,70 +4,67 @@ nav_title: Logistics
...
 
 * You will need to have included this request in your application.
 * You will be required to provide documentation of travel expenses (such as transportation and lodging receipts) and to fill out a reimbursement form after the workshop ends.
 * You must attend the entirety of the workshop.
+ 
+## Health and Safety
+
+Alex’s Lemonade Stand Foundation’s Childhood Cancer Data Lab will comply with any governmental requirements related to health and safety. 
+We encourage workshop participants to adhere to US governmental guidance. 
+You may wish to review the [CDC COVID-19 by County guidance](https://www.cdc.gov/coronavirus/2019-ncov/your-health/covid-by-county.html) and the [Philadelphia, Pennsylvania COVID-19 community level](https://covid.cdc.gov/covid-data-tracker/#county-view?list_select_state=all_states&list_select_county=all_counties&data-type=Vaccinations) when planning for workshop attendance.
```
Comment (sjspielman) [inline docs/participant_information.md:67]: Hm, the latter is a vaccination rate link. How about this one?

```suggestion
You may wish to review the [CDC COVID-19 by County guidance](https://www.cdc.gov/coronavirus/2019-ncov/your-health/covid-by-county.html) and the [Philadelphia, Pennsylvania COVID-19 community level](https://covid.cdc.gov/covid-data-tracker/#trends_weeklyhospitaladmissions_select_00) when planning for workshop attendance.
```

File: docs/setup_instructions/setup_overview.md [hunk 4]
```diff
@@ -18,7 +18,4 @@ Finally, you will also need to create a [GitHub](https://github.com) account if
 - [GitHub and GitKraken setup](github_gitkraken_setup_instructions.md)
 
 
-If you need any assistance with the installation, do not hesitate to reach out to the Data Lab team by email, or, even better, in the [Cancer Data Science Slack](http://ccdatalab.org/slack) (you will find us in the #reproducible-research-practices channel)!
-
-
-
+If you need any assistance with the installation, do not hesitate to reach out to the Data Lab team by email, or, even better, in the [Cancer Data Science Slack](http://ccdatalab.org/slack). We will add you to a training specific channel after you join the Slack community.
```
Comment (sjspielman) [inline docs/setup_instructions/setup_overview.md:21]:
```suggestion
If you need any assistance with the installation, do not hesitate to reach out to the Data Lab team by email, or, even better, in the [Cancer Data Science Slack](http://ccdatalab.org/slack). We will add you to a training-specific channel after you join the Slack community.
```

**Review by sjspielman — COMMENTED** [review]
This all looks fine to me! Left a couple text suggestions and one comment just for fun 😄 
I'm just holding off on approving because I want to check back with the issue this is addressing (see my other comment - link isn't working?), but I don't anticipate really having much else to say!

File: docs/participant_information.md (hunk 2 above)
Comment (sjspielman) [reply docs/participant_information.md:35]: Wait they do catering?! https://order.toasttab.com/online/manakeesh-cafe-bakery-grill#dfca412c4-d782-4a16-9562-1531165a4f18d63ce381c-e43e-4ff9-8b26-85d64733b21b
please therefore consider this my vote to get them to cater something at some point in the future ;)

**Comment by Jen-OMalley** [comment]
> @Jen-OMalley the link to the `training-admin` issue doesn't appear to be working - can you comment with the right link? Thanks!

Oops! It should be this issue. https://github.com/AlexsLemonade/training-admin/issues/1016

**Review by sjspielman — APPROVED** [review]
Looks good for the issue, so approving now assuming you'll catch my reviews! lmk if anything is unclear or you disagree with something!

**Pull request merged**